showup/models.py

```diff
@@ -26,21 +36,14 @@ def __str__(self):
         )
 
 
-class Genre(models.Model):
-    genre = models.TextField()
-
-    def __str__(self):
-        return self.genre
-
-
 class CustomUser(AbstractUser):
     date_of_birth = models.DateField(null=True, blank=True)
     gender = models.CharField(max_length=255, blank=True)
     interested = models.ManyToManyField(Concert, related_name="interested", blank=True)
     going = models.ManyToManyField(Concert, related_name="going", blank=True)
     bio = models.TextField(max_length=500, default="", blank=True)
     swipes = models.ManyToManyField("self", through="Swipe", symmetrical=False)
-    genres = models.ManyToManyField(Genre, related_name="genres", blank=True)
+    genres = models.ManyToManyField(Genre, related_name="fav_genres", blank=True)
```

Comment:
did it need to be renamed?

Reply:
Yes because otherwise there would be a conflict with the related name of the genres field in the Concert model

Reply:
ok thx